externs/shaka/player.js

```diff
@@ -928,8 +928,9 @@ shaka.extern.ManifestConfiguration;
  * @property {boolean} useNativeHlsOnSafari
  *   Desktop Safari has both MediaSource and their native HLS implementation.
  *   Depending on the application's needs, it may prefer one over the other.
- *   Examples: FairPlay is only supported via Safari's native HLS, but it
- *   doesn't have an API for selecting specific tracks.
+ *   Warning: Where single-key HLS Fairplay streams work fine, multi-keys
```

Comment:
It's not clear if this warning is meant to apply to when native HLS is enabled or disabled.